src/components/ui/LabelTag.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components/macro'
+
+const LabelTag = styled.span`
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  border-radius: 4px;
+  height: 15px;
+  padding: 2px;
+  font-size: 9px;
+  font-weight: bold;
+  color: ${({ theme }) => theme.text};
+  background-color: #dadada;
```

Comment:
Ask siraj what color this approximates to in theme, change it to that, or add the new color to theme

Let's be sure to always do this in PRs from now on -- don't hardcode colors.

Reply:
Gotchu, won't hardcode colors ever again sir

Reply:
LOL nw